src/Components/Share/ShareModal.js

```diff
@@ -0,0 +1,32 @@
+import React, { useState, useEffect } from "react";
+
+const ShareModal = ({ url, id }) => {
+  const urlInputId = `${id}-share-url`;
+  const [copied, setCopied] = useState(false);
+
+  const shareButtonClick = () => {
+    const shareUrl = document.getElementById(urlInputId);
+    shareUrl.select();
+    setCopied(true);
```

Comment:
By setting state we unfortunately leave the button saying 'Copied' for the rest of the app (it's been rendered and despite being hidden when the modal collapses the component is not unmounted.  Some options:
- Actually unmount as it is hidden
- Set a timer to reset back to copied-false after a delay
- Have it appear as a bit of disappearing text instead